src/tensors/abstracttensor.jl

```diff
@@ -185,15 +185,16 @@ function adjointtensorindices(t::AbstractTensorMap, p::Index2Tuple)
 end
 
 @doc """
-    blocks(t::AbstractTensorMap)
+    blocks(t::AbstractTensorMap) -> Dict{<:Sector,<:DenseMatrix}
```

Comment:
```suggestion
    blocks(t::AbstractTensorMap) -> SectorDict{<:Sector,<:DenseMatrix}
```
I think this actually returns a `SortedVectorDict`, which is aliased by `SectorDict`. The name is probably clear enough to warrant specifying this detail?